content/en/apps/guides/forms/rdtoolkit-integration.md

```diff
@@ -0,0 +1,219 @@
+
+Optional fields:
+* `rdtoolkit_state` | Text
+    * The RDT status.
+    * Defined in the `data` group level.
+    * Auto filled by CHT integration.
+* `rdtoolkit_time_started` | Text
+    * The date and time when RDT started.
+    * Defined in the `data` group level.
+    * Auto filled by CHT integration.
+* `rdtoolkit_time_resolved` | Text
+    * The date and time when the RDT result is ready to capture.
+    * Defined in the `data` group level.
+    * Auto filled by CHT integration.
+* `rdtoolkit_time_read` | Text
+    * The date and time when RDT result was captured.
+    * Defined in the `data` group level.
+    * Auto filled by CHT integration.
+
+### Sample form for capturing RDT results
+![Capturing RDT result form](rdtoolkit-capture-form.png)
+
+## Create a task
+A [task](https://docs.communityhealthtoolkit.org/apps/features/tasks/) is added when a new provisioning RDT form is created, and it will be resolved when a capture RDT result form is saved. 
+The task needs to copy the `rdtoolkit_session_id` from the provision form to the capture form in order to retrieve RDT results from the RDToolkit app. 
+
+The following is an example of a task implementation:
+
+```javascript
+{
+  name: 'rdtoolkit-capture-results',
+  icon: 'icon-follow-up',
+  title: 'task.rdtoolkit.capture.title',
+  appliesTo: 'reports',
+  appliesToType: ['rdtoolkit-provision'], // The form associated with the creation of tasks.
+  appliesIf: (contact, report) => {
+    return !!(getField(report, 'data.__patient_id') && getField(report, 'data.rdtoolkit_session_id'));
+  },
+  resolvedIf: (contact, report, event, dueDate) => {
+    if (!contact.reports) {
+      return false;
+    }
+
+    const captureReport = contact.reports.find(reportDoc => {
+      if (reportDoc.form !== 'rdtoolkit-capture') {
+        return false;
+      }
+      return getField(reportDoc, 'rdtoolkit_session_id') === getField(report, 'data.rdtoolkit_session_id');
+    });
+
+    if (!captureReport || !getField(captureReport, 'data.rdtoolkit_results')) {
+      return false;
+    }
+
+    const startTime = Math.max(addDays(dueDate, -event.start).getTime(), report.reported_date + 1);
+    const endTime = addDays(dueDate, event.end + 1).getTime();
+
+    return isFormArraySubmittedInWindow(contact.reports, ['rdtoolkit-capture'], startTime, endTime);
+  },
+  actions: [
+    {
+      type: 'report',
+      form: 'rdtoolkit-capture',
+      modifyContent: (content, contact, report) => {
+        content.rdtoolkit_session_id = getField(report, 'data.rdtoolkit_session_id'); // Copy rdtoolkit_session_id from provision form to capture form.
+      }
+    }
+  ],
+  events: [
+    {
+      id: 'rdtoolkit-capture-event',
+      start: 1,
+      end: 2,
+      days: 1
+    }
+  ]
+}
+```
+
+## Add translations
+Add translations by following the [CHT guide](https://docs.communityhealthtoolkit.org/apps/reference/translations/). The following is an example of translation labels:
```

Comment:
Make sure to use the ref or relref function to [avoid broken links](https://docs.communityhealthtoolkit.org/contribute/docs/style-guide/#avoid-broken-links).

Reply:
Nice! I'll fix it